packages/block-editor/src/components/block-mover/index.js

```diff
@@ -66,6 +66,8 @@ export class BlockMover extends Component {
 					aria-disabled={ isFirst }
 					onFocus={ this.onFocus }
 					onBlur={ this.onBlur }
+					onMouseEnter={ this.onFocus }
```

Comment:
Could you explain this change?

Reply:
This PR is based on @afercia 's [PR](https://github.com/WordPress/gutenberg/pull/8147/files#diff-65de31d7ac3309c01953b37ae9c2a7eeR70)
Maybe @afercia might help here since I've iterated over his initial work?

Reply:
Honestly I can't remember why I made that change. That was one year ago 😬 Sorry!

Reply:
@aduth @afercia, FYI - I have reverted these changes.